tests/testcases/core/services/commands/CommandBusTest.php

```diff
@@ -38,8 +38,8 @@ public function setUp()
         // need to override the existing alias for the CommandHandlerManagerInterface
         // or else the REAL class will still get used
         EE_Dependency_Map::instance()->add_alias(
-            'EventEspresso\core\services\commands\CommandHandlerManagerInterface',
-            'EventEspresso\tests\mocks\core\services\commands\CommandHandlerManagerMock'
+            'EventEspresso\tests\mocks\core\services\commands\CommandHandlerManagerMock',
+            'EventEspresso\core\services\commands\CommandHandlerManagerInterface'
```

Comment:
why was this changed around?  Does it mean the behaviour of `EE_Dependency_Map::instance()->add_alias` has changed?  Does that have a wider impact than the original purpose for this pull and thus needs more specific testing coverage?

Reply:
This was wrong. This was basically saying:

```
substitute all requests for EventEspresso\tests\mocks\core\services\commands\CommandHandlerManagerMock
with an instance of EventEspresso\core\services\commands\CommandHandlerManagerInterface
```

which of course is impossible and should have been the other way around.

This error was detected by the addition of the `is_subclass_of()` check to `core/services/loaders/ClassInterfaceCache.php`